spec/features/elibility_confirmation_spec.rb

```diff
@@ -11,27 +11,29 @@
     it 'shows an eligibility confiration overlay that can be cleared to reveal the form' do
       visit new_page_path(item_id: '1234', origin: 'GREEN', origin_location: 'STACKS')
 
-      expect(page).to have_css('#new_request', visible: false)
-      expect(page).to have_css('#eligibility-confirm-overlay', visible: true)
+      expect(page).to have_css('#new_request', obscured: true)
```

Comment:
This is something handy that I just learned about today (only available in the JS driver)